docs/development.md

```diff
@@ -37,7 +63,7 @@ python3 -m pip install pip-tools
 python3 -m piptools compile --output-file=requirements.txt requirements.in
```

Comment:
You no longer need to run piptools this way. You can now run this make target: `python-preproc-requirements`